crates/driver/src/tests/setup/mod.rs

```diff
@@ -492,6 +506,10 @@ pub struct Setup {
     mempools: Vec<Mempool>,
     /// Extra configuration for the RPC node
     rpc_args: Vec<String>,
+    /// Whether the solver sends the solution as a JIT order
+    jit_order: Option<Order>,
+    /// List of surplus capturing JIT-order owners
+    surplus_capturing_jit_order_owners: Vec<H160>,
```

Comment:
Should no longer be needed if the JIT orders just get signed with a dedicated surplus capturing JIT order owner key.

Reply:
In this particular case, these addresses are the ones we need to send from the `autopilot` as the surplus capturing JIT order owners (since it is no longer set in the configuration of the driver).